astropy/io/fits/scripts/fitsdiff.py

```diff
@@ -252,7 +266,8 @@ def main():
         opts.ignore_keywords = []
         opts.ignore_comments = []
         opts.ignore_fields = []
-        opts.tolerance = 0.0
```

Comment:
Not sure, but why not `= None`? All the other defaults are explicitly listed here.